index.html

```diff
@@ -75,33 +87,146 @@ <h1 class="title">
 
     <section class="works">
       <h2 class="my-recent">My Recent Works</h2>
-      <hr />
-      <div class="placeholder">
-        <img src="images/Img Placeholder.svg" alt="Image Placeholder" />
+      <hr class="hori-break1" />
+      <div class="clearfix">
+        <div class="placeholder">
+          <img src="images/Img Placeholder.svg" alt="Image Placeholder" />
+        </div>
+        <img
+          class="placeholder2"
+          src="images/Img Placeholder-desktop.svg"
+          alt=""
+        />
+        <div class="stories">
+          <p>Multi-Post Stories</p>
+        </div>
+        <p class="daily">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+        <ul class="lruby">
+          <a class="block" href="#">
+            <li>css</li>
+          </a>
+          <a class="block" href="#">
+            <li>html</li>
+          </a>
+          <a class="block" href="#">
+            <li>bootstrap</li>
+          </a>
+          <a class="block" href="#">
+            <li>ruby</li>
+          </a>
+        </ul>
+        <a class="see-project" href="#"> See Project </a>
       </div>
-      <div class="stories">
-        <p>Multi-Post Stories</p>
+    </section>
+
+    <section class="project-cards-desktop">
+      <div class="nothing">
+        <button class="project-button-desktop" href="#">See Project</button>
+      </div>
+      <div class="profesion-desktop">
+        <h2 class="heading-content-desktop">
+          Data Dashboard <br />
+          Healthcare
+        </h2>
+        <p class="p-content-desktop">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="p-language-desktop">
+          <a class="block-two-desktop" href="#">
+            <li>html</li>
+          </a>
+          <a class="block-two-desktop" href="#">
+            <li>bootstrap</li>
+          </a>
+          <a class="block-two-desktop" href="#">
+            <li>ruby</li>
+          </a>
+        </ul>
```

Comment:
- [ ] I think you should align the content of your project cards descriptions properly to the left to match the Figma design you chose.
You can check the menu-items CSS properties of Figma design😄 

| Figma Design | Your Design |
| ------------- | ------------- |
|![image](https://user-images.githubusercontent.com/63473584/193130596-0b13c30b-11f3-4550-86cc-b03290ba678b.png)  |![image](https://user-images.githubusercontent.com/63473584/193130691-42b05495-b0a0-41b0-8c13-fa0df203da36.png)  |

Reply:
@vickymarz 
I adjusted the alignment of my project cards descriptions in the CSS file